src/guide/single-file-component.md

```diff
@@ -1,67 +1,67 @@
+- [Resaltado de sintaxis completa](https://github.com/vuejs/awesome-vue#source-code-editing)
+- [Módulos CommonJS](https://webpack.js.org/concepts/modules/#what-is-a-webpack-module)
+- [CSS con alcance por componente](https://vue-loader.vuejs.org/en/features/scoped-css.html)
 
-As promised, we can also use preprocessors such as Pug, Babel (with ES2015 modules), and Stylus for cleaner and more feature-rich components.
+Como se prometió, también podemos utilizar preprocesadores como Pug, Babel (con módulos ES2015), y Stylus para componentes más claros y completos en características.
 
 <a href="https://codesandbox.io/s/vue-single-file-component-with-pre-processors-mr3ik?file=/src/App.vue" target="_blank" rel="noopener noreferrer"><img src="/images/sfc-with-preprocessors.png" width="563" alt="Single-file component with pre-processors example (click for code as text)" style="display: block; margin: 15px auto; max-width: 100%"></a>
 
-These specific languages are only examples. You could as easily use TypeScript, SCSS, PostCSS, or whatever other preprocessors that help you be productive. If using Webpack with `vue-loader`, it also has first-class support for CSS Modules.
+Estos lenguajes específicos son únicamente ejemplos. Puede utilizar fácilmente TypeScript, SCSS, PostCSS, o cualquier otro preprocesador que le ayude a ser productivo. Si utiliza Webpack con `vue-loader`, también tiene soporte de primera clase para Módulos CSS.
 
-### What About Separation of Concerns?
+### ¿Qué hay sobre la Separación de Responsabilidades?
 
-One important thing to note is that **separation of concerns is not equal to separation of file types.** In modern UI development, we have found that instead of dividing the codebase into three huge layers that interweave with one another, it makes much more sense to divide them into loosely-coupled components and compose them. Inside a component, its template, logic and styles are inherently coupled, and collocating them actually makes the component more cohesive and maintainable.
+Una cosa importante a notar es que **la separación de responsabilidades no es igual a la separación de tipos de archivo.** En el desarrollo de interfaces de usuario modernas, hemos encontrado que en vez de dividir el código base en tres grandes capas que se entrelazan una con otra, tiene mucho más sentido dividirlas en componentes con bajo acoplamiento y componer los mismos. Dentro de un componente, su plantilla, lógica y estilos están acoplados inherentemente, y colocándolos de hecho hace al componente más cohesivo y mantenible.
 
-Even if you don't like the idea of Single-File Components, you can still leverage its hot-reloading and pre-compilation features by separating your JavaScript and CSS into separate files:
+Incluso si no le gusta la idea de Componentes de un Solo Archivo, todavía puede aprovechar sus características de recarga en caliente y precompilación separando su JavaScript y CSS en archivos separados:
```

Comment:
Lets leave `hot-realoding` as is
```suggestion
Incluso si no le gusta la idea de Componentes de un Solo Archivo, todavía puede aprovechar sus características de _hot-realoading_ y precompilación separando su JavaScript y CSS en archivos separados:
```

Reply:
Fixed `hot-realoding` typo. Changed to `hot-reloading`.